src/components/Q31StarPoint/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { useMemo } from 'react';
+import { AiFillStar } from 'react-icons/ai';
+
+interface IQ31StarPointProp {
+    amount: number;
+}
+
+export const Q31StarPoint: React.FC<IQ31StarPointProp> = ({ amount }) => {
+    const point = useMemo(
+        () => [...Array(5)].map((_, index) => index < amount),
+        [amount]
+    );
+
+    return (
+        <div className="mt-5">
+            <ul className="align-items-center p-0 row row-cols-3 pb-3">
+                <li className="d-flex align-items-center list-unstyled col pb-4">
```

Comment:
1) Um componente não deve nunca ter margins e paddings externos que influenciem outros elementos ao redor dele. Cada componente deve ser reutilizável em qualquer situação. Pode remover `mt-5`, `pb-3` e `pb-4`.

2) `row` e `col` só devem ser utilizados quando for dividir o layout em grids.

3) `ul` e `li` não fazem o menor sentido nessa situação. Você consegue resolver isso com 2 divs apenas.